www/front_src/src/Resources/translatedLabels.ts

```diff
@@ -200,3 +200,5 @@ export const labelForward = 'Forward';
 export const labelBackward = 'Backward';
 export const labelEndDateGreaterThanStartDate =
   'The end date must be greater than the start date';
+export const labelGraphOptions = 'Graph options';
+export const labelToggleTooltipValues = 'Toggle tooltip values';
```

Comment:
```suggestion
export const labelToggleTooltipValues = 'Display metric values in legend';
```

What do you think? so that it’s a bit more clear. Then we would need to reverse the value

Reply:
Mmmhhh.... I'm not sure to understand. That means, we toggle metric values display in legend